test/automation/src/positron/positronNewProjectWizard.ts

```diff
@@ -247,24 +248,48 @@ class ProjectWizardPythonConfigurationStep {
 			);
 		}
 
-		// Open the dropdowns
+		// Open the interpreter dropdown.
 		await this.interpreterDropdown.click();
 
 		// Try to find the interpreterPath in the dropdown and click the entry if found
 		try {
 			await this.code.waitForElement(PROJECT_WIZARD_DROPDOWN_POPUP_ITEMS);
-			await this.code.driver
-				.getLocator(
-					`${PROJECT_WIZARD_DROPDOWN_POPUP_ITEMS} div.dropdown-entry-subtitle`
-				)
-				.getByText(interpreterPath)
-				.click();
-			return;
 		} catch (error) {
 			throw new Error(
-				`Could not find interpreter path in project wizard dropdown: ${error}`
+				`Wait for element ${PROJECT_WIZARD_DROPDOWN_POPUP_ITEMS} failed: ${error}`
 			);
 		}
+
+		// Get all the dropdown entry subtitles and build a comma-separated string of them for
+		// logging purposes.
+		const dropdownEntrySubtitleLocators = await this.code.driver
+			.getLocator(
+				`${PROJECT_WIZARD_DROPDOWN_POPUP_ITEMS} div.dropdown-entry-subtitle`
+			).all();
+		let subtitles = '';
+		for (let i = 0; i < dropdownEntrySubtitleLocators.length; i++) {
+			if (i) {
+				subtitles += ',';
+			}
+			subtitles += await dropdownEntrySubtitleLocators[i].innerText();
+		}
```

Comment:
ignoreable alternative:

```suggestion
		const dropdownEntrySubtitles = dropdownEntrySubtitleLocators.map
			(async (locator) => await locator.innerText());
		const subtitles = (await Promise.all(dropdownEntrySubtitles)).join(', ');
```